### PR TITLE
changed to Material IO Css for Table

### PR DESCRIPTION
**Pull request by scottmillion**

### CODE REVIEW

File: src/components/CoinListChart/CoinListChart.js [hunk 1]
```diff
@@ -41,18 +41,22 @@ const options = {
...
-      {
-        fill: false,
-      },
-    ],
+  const data = (canvas) => {
+    const data = {
+      labels: new Array(props.prices.length).fill(''),
+      datasets: [
+        {
+          fill: false,
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinListChart/CoinListChart.js:49]: data: props.prices,
borderColor :`${
      props.sevenDayChange < 0 ? '#FE1040' : '#00FC2A'
    }`

File: src/components/CoinListChart/CoinListChart.js [hunk 2]
```diff
@@ -41,18 +41,22 @@ const options = {
...
+        },
+      ],
+    }
+    data.datasets[0].data = props.prices
+    data.datasets[0].borderColor = `${
+      props.sevenDayChange < 0 ? '#FE1040' : '#00FC2A'
+    }`
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinListChart/CoinListChart.js:56]: no need to do this

File: src/store/allCoins/index.js [hunk 3]
```diff
@@ -96,39 +134,28 @@ function allCoinsReducer(state = initialState, action) {
...
-      }
+      const configProperty = Object.entries(state.config).find(
+        (ele) => ele[1].sortBy === action.payload,
+      )[0]
 
       return {
         ...state,
         coinsData: state.coinsData.sort((a, b) =>
-          state[sortAsc]
+          state.config[configProperty]['sortByAsc']
```
Comment (yoloOnTheBattlefield) [inline src/store/allCoins/index.js:144]: state.config[configProperty].sortByAsc

File: src/store/allCoins/index.js [hunk 4]
```diff
@@ -96,39 +134,28 @@ function allCoinsReducer(state = initialState, action) {
...
             ? a[action.payload] < b[action.payload]
```
Comment (yoloOnTheBattlefield) [inline src/store/allCoins/index.js:145]: maybe put this in a variable so you don't repeat yourself

const { payload } = action;
a[payload] < b[payload]

File: src/store/allCoins/index.js [hunk 5]
```diff
@@ -96,39 +134,28 @@ function allCoinsReducer(state = initialState, action) {
...
             : b[action.payload] < a[action.payload]
             ? -1
             : 1,
         ),
-        [sortAsc]: !state[sortAsc],
+        config: {
+          ...state.config,
+          [configProperty]: {
+            ...state.config[configProperty],
+            sortByAsc: !state.config[configProperty]['sortByAsc'],
```
Comment (yoloOnTheBattlefield) [inline src/store/allCoins/index.js:156]: see what I did before

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/CoinListChart/CoinListChart.js (hunk 1 above)
Comment (scottmillion) [reply src/components/CoinListChart/CoinListChart.js:49]: Done

File: src/components/CoinListChart/CoinListChart.js (hunk 2 above)
Comment (scottmillion) [reply src/components/CoinListChart/CoinListChart.js:56]: I see it. Removed.

File: src/store/allCoins/index.js (hunk 3 above)
Comment (scottmillion) [reply src/store/allCoins/index.js:144]: Done

File: src/store/allCoins/index.js (hunk 4 above)
Comment (scottmillion) [reply src/store/allCoins/index.js:145]: Done

File: src/store/allCoins/index.js (hunk 5 above)
Comment (scottmillion) [reply src/store/allCoins/index.js:156]: Done

**Pull request merged**